font/src/ft/mod.rs

```diff
@@ -32,28 +34,28 @@ use super::{
     Style, Weight,
 };
 
+#[derive(Debug, PartialEq, Copy, Clone)]
 struct FixedSize {
     pixelsize: f64,
 }
```

Comment:
Unrelated to this PR directly, but since there's a change to this: Why is this a dedicated struct? Seems like we could just as well use a `f64` directly?